Sources/ApiVideoPlayer/ApiVideoPlayerController.swift

```diff
@@ -505,10 +539,52 @@ public class ApiVideoPlayerController: NSObject {
                 case let .failure(error): print("analytics error on resume event: \(error)")
                 }
             }
+            self.infoNowPlaying?.play(currentTime: self.currentTime, currentRate: self.avPlayer.rate)
         }
+        #if !os(macOS)
+        try? AVAudioSession.sharedInstance().setCategory(AVAudioSession.Category.playback)
+        try? AVAudioSession.sharedInstance().setActive(true)
+        #endif
         self.multicastDelegate.didPlay()
     }
 
+    #if !os(macOS)
+    func remoteControlEventReceived(with event: UIEvent?) {
+        if let event = event {
+            switch event.subtype {
+            case .remoteControlPlay:
+                // Handle play remote control event
+                self.play()
+                self.infoNowPlaying?.play(currentTime: self.currentTime, currentRate: self.avPlayer.rate)
```

Comment:
Do we have to explicitely call this althought it will be called in `doPlayAction` callback?